ksz8863/examples/simple_switch/main/simple_switch_main.c

```diff
@@ -313,20 +316,28 @@ void app_main(void)
     ESP_ERROR_CHECK(esp_event_handler_register(ETH_EVENT, ESP_EVENT_ANY_ID, &eth_event_handler, &host_eth_handle));
     ESP_ERROR_CHECK(esp_event_handler_register(IP_EVENT, IP_EVENT_ETH_GOT_IP, &got_ip_event_handler, NULL));
 
+    // We need to wait for IP to be obtained because starting l2tap tasks may impede normal function of DHCP client
```

Comment:
Could you please elaborate why? It may affect the DHCP before when you filtered IP traffic to L2TAP. Now, it should be fine I guess. Please verify. Also note that if this was a case, you can see that filtering IP traffic to L2TAP was "evil".